k8s/sealedthumbor-secret.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: thumbor-secrets
+  namespace: cbr57382
```

Comment:
Does this stop this sealed secret working in the image-service namespace?

Reply:
Sorry this wasn't completely up to date, the secret running in the image-service namespace has the namespace field set to "image-service"